extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -146,18 +147,49 @@ function SidebarWithRef(
   useLayoutEffect(() => {
     const sidebarElement = sidebarRef.current;
     const backdropElement = backdropRef.current;
-    if (!sidebarElement || !backdropElement) {
-      return;
-    }
-
-    if (!side) {
+    if (!side || !sidebarElement || !backdropElement) {
       return;
     }
 
     let sidebarAnimation;
     let backdropAnimation;
+    let keydownCallback;
+
+    const postInvisibleAnim = () => {
+      if (onAfterCloseRef.current) {
+        onAfterCloseRef.current();
+      }
+      sidebarAnimation = null;
+      backdropAnimation = null;
+      setMounted(false);
+    };
+    const documentElement =
+      sidebarElement.ownerDocument &&
+      sidebarElement.ownerDocument.documentElement;
+
     // "Make Visible" Animation
     if (opened) {
+      if (documentElement) {
```

Comment:
This should be a separate `useEffect` hook on `[open]`. There's really little value in sharing the effects.

Reply:
Updated in the latest.